graphql/src/subscription/mod.rs

```diff
@@ -222,6 +223,7 @@ where
         variable_values,
         deadline: timeout.map(|t| Instant::now() + t),
         max_first,
+        block: BLOCK_NUMBER_MAX,
```

Comment:
Subscribing to specific blocks is funny... at least for historic blocks that data is never going to change. There may be situations where it makes sense though (future block heights maybe)?

Reply:
Not sure if it ever makes sense to subscribe at a specific block height, and `BLOCK_NUMBER_MAX` means 'give me whatever the latest is' which is what we want for subscriptions.